mkdocs.yml

```diff
@@ -156,6 +156,7 @@ nav:
       - users/install/index.md
       - users/install/docker-installation.md
       - users/install/ddev-installation.md
+      - users/install/ddev-upgrade.md
       - 'Configuring Browsers': users/install/configuring-browsers.md
       - 'Upgrading': users/install/ddev-upgrade.md
```

Comment:
If this section is moved up, should be removed here
```suggestion
```

Reply:
Thanks. I couldn't actually see that it was there.  Now I can see that it is. Will just revert the change.